classes/components/listPanels/DoiListPanel.php

```diff
@@ -65,6 +65,9 @@ protected function setAppConfig(array &$config): void
         // Provide required locale keys
         $request = Application::get()->getRequest();
         $templateMgr = TemplateManager::getManager($request);
-        $templateMgr->setLocaleKeys(['submission.monograph']);
+        $templateMgr->setLocaleKeys([
```

Comment:
I can't remember if we need to manually set the locale keys for Vue components anymore. I know this is true in `main` but @jardakotesovec can confirm if this is also true in `stable-3_4_0`.

Reply:
I double checked with @jardakotesovec and yes, in 3.4. it is still either with setLocaleKeys or passing it as props...